nx.json

```diff
@@ -12,5 +13,38 @@
       "outputs": ["{projectRoot}/dist"]
     }
   },
+  "release": {
+    "projectsRelationship": "fixed",
+    "groups": {
+      "grafanaPackages": {
+        "projects": [
+          "@grafana/data",
+          "@grafana/e2e",
+          "@grafana/e2e-selectors",
+          "@grafana/flamegraph",
+          "@grafana/prometheus",
+          "@grafana/runtime",
+          "@grafana/schema",
+          "@grafana/ui"
+        ]
+      },
+      "privatePackages": {
+        "projects": ["@grafana/sql", "@grafana/o11y-ds-frontend"]
```

Comment:
Should `@grafana/saga-icons` be in this group too?

Reply:
Hmm... I don't think it should sit in this group. More likely we should assign it it's own group so it can be bumped and released individually from the other packages (most likely via a gh workflow).